src/actions/graph.js

```diff
@@ -105,24 +94,23 @@ const isChonky = (chonkyNodeAmount, nodesNo, edgesNo) => {
  * @param {Object} graphState A subset of main state
  * @return {function} A promise that resolves when the calcuation is done
  */
-export function calculateGraph(graphState, customChonkyLimit) {
+export function calculateGraph(graphState, customLargeLimit) {
```

Comment:
Worth considering a default param here to clean this up potentially e.g.

`export function calculateGraph(graphState, displayThreshold=largeGraphThreshold)` that

Reply:
good suggestion, I have assigned the default param later down the code, but I can move it up here 👍

Reply:
updated to reflect this in latest commit